src/main/java/Task.java

```diff
@@ -0,0 +1,46 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public Task(String description, boolean isDone) {
+        this.description = description;
+        this.isDone = isDone;
+    }
+
+    public boolean markAsDone() {
+        this.isDone = true;
+        return true;
+    }
+
+    public String getType() {
+        if(this instanceof ToDo) {
+            return "T";
+        } else if(this instanceof Deadline) {
+            return "D";
+        } else {
+            return "E";
+        }
+    }
+
+    public String getDescription() {
+        return this.description;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "\u2713" : "\u2718");
+    }
+
+    public boolean getIsDone() {
```

Comment:
Perhaps a better name would be isTaskDone?

Reply:
I agree! Thank you.